api/src/main/java/de/oliver/fancyholograms/api/utils/GeyserChecker.java

```diff
@@ -0,0 +1,17 @@
+package de.oliver.fancyholograms.api.utils;
+
+import org.bukkit.entity.Player;
+import org.geysermc.geyser.api.GeyserApi;
+
+public class GeyserChecker {
+
+    public static boolean isGeyserPlayer(Player player) {
+        if (player == null) return false;
+
+        try {
+            return GeyserApi.api().connectionByUuid(player.getUniqueId()) != null;
```

Comment:
As mentioned above, I recommend using Floodgate instead:
```suggestion
            return FloodgateApi.getInstance().isFloodgatePlayer(player.getUniqueId());
```